client/src/components/navigation-bar.js

```diff
@@ -3,19 +3,57 @@ import { Nav, Navbar } from 'react-bootstrap';
 import { Link } from 'react-router-dom';
 import NavigationItem from './shared/navigation/navigation-item';
 import info from './info/navigation';
+import lognavigation from './info/lognavigation';
 
 class NavigationBar extends Component {
-  render() {
-    const className = 'mr-3';
-    const navBarContent = info.map((e) => (
+  // eslint-disable-next-line consistent-return
+  constructor(props) {
+    super(props);
+    this.state = {
+      isAuthentificated: false,
+    };
+  }
+
+  componentDidMount() {
+    if (localStorage.getItem('Bearer ') != null) {
+      this.setState({ isAuthentificated: true });
+    }
+    window.setInterval(() => {
+      if (localStorage.getItem('Bearer ') != null) {
+        this.setState({ isAuthentificated: true });
+      } else {
+        this.setState({ isAuthentificated: false });
+      }
+    }, 500);
+  }
```

Comment:
I think this method (`componentDidMount()`) should be in **[app.js](https://github.com/Social-projects-Rivne/Lunch_Time/blob/dev/client/src/app.js)** file. 

Firstly, this is not NavBar's responsibility (I mean checking `isAuthentificated`). 

Secondly, this will work every next app page switch and will check if `isAuthentificated` just when new app page is entered. We will not need `window.setInterval()`.

Also some syntax:
- use camelCase for `logNavigation` (starting from line 6) and better `navigationForLoggedIn`;
- use renderFunctio**n** (starting from line 31);

Optional:
- we can use in NavBar's `componentDidMount()` if-else statement due to `isAuthentificated` value, store links and components in `set` and render only one element (`set`) once each time if `isAuthentificated` value is changed;